### DISCUSSION TITLE
Control Horizon is assumed to be the same as the Prediction Horizon

### DISCUSSION THREAD

**Discussion by SaeidBayat** [do-mpc/do-mpc#402] [Q&A]
Hi,
Can we change the control horizon? It seems that in this library, the control horizon is assumed to be the same as the prediction horizon, right?

**Comment by Mrfeli01** [comment]
Dear @saeidb71,
in do-mpc we use the same number of steps for the control horizon and the prediction horizon.
Best
Mrfeli
